cdap-docs/_common/common-build.sh

```diff
@@ -354,10 +264,48 @@ function download_file() {
   test_an_include ${md5_hash} ${target}
 }
 
+function build_license_pdfs() {
+  set_version
+  cd ${SCRIPT_PATH}
+  PROJECT_VERSION_TRIMMED=${PROJECT_VERSION%%-SNAPSHOT*}
+  rm -rf ${SCRIPT_PATH}/${LICENSES_PDF}
+  mkdir ${SCRIPT_PATH}/${LICENSES_PDF}
+  # paths are relative to location of $DOC_GEN_PY script
+  LIC_PDF="../../../${REFERENCE}/${LICENSES_PDF}"
+  LIC_RST="../${REFERENCE}/source/${LICENSES}"
```

Comment:
Does it make more sense to make these absolute PATHs? This allows us to move the `$DOC_GEN_PY` script and not have to find and edit these relative PATHs, that would otherwise be difficult to correlate with the script move.